plugins/Ecommerce/Controller.php

```diff
@@ -76,11 +76,21 @@ public function getConversionsOverview()
     {
         $view = new View('@Ecommerce/conversionOverview');
         $idGoal = Common::getRequestVar('idGoal', null, 'string');
-
-        $goalMetrics = Request::processRequest('Goals.get', array('idGoal' => $idGoal, 'filter_limit' => '-1'), $default = []);
+        $idSite = Common::getRequestVar('idSite', null, 'int');
```

Comment:
here we could probably use `$this->idSite` but not needed. Should be all good to merge!

Reply:
changed that